src/core/sd_functions.cpp

```diff
@@ -592,19 +619,19 @@ String loopSD(FS &fs, bool filePicker, String allowed_ext) {
       if(checkSelPress())
       {
         // Definição da matriz "Options"
-        if(fileList[index][2]=="folder") {
+        if(fileList[index].folder==true && fileList[index].operation==false) {
```

Comment:
if FileList.folder and FileList.operation are both booleans, comparing it to a boolean makes it redundant. You could do `if(fileList[index].folder && ! fileList[index].operation)`. Same for other places

Reply:
> Explicit is better than implicit.

https://en.m.wikipedia.org/wiki/Zen_of_Python#Principles